patch_api/models/__init__.py

```diff
@@ -16,6 +16,23 @@
 
 # import models into model package
 from patch_api.models.allocation import Allocation
+from patch_api.models.create_bitcoin_estimate_request import (
+    CreateBitcoinEstimateRequest,
+)
```

Comment:
Those models were previously ignored by our codegen. 

Proposing to stay as close as possible from the standard output of https://github.com/OpenAPITools/openapi-generator and stop ignoring them. 

AFAIK, this has no impact.